src/PostgREST/App.hs

```diff
@@ -261,38 +248,55 @@ app dbStructure conf apiRequest =
   filterCol _ _ _ =  False
   allPrKeys = dbPrimaryKeys dbStructure
   allOrigins = ("Access-Control-Allow-Origin", "*") :: Header
-  jsonH = ctToHeader CTApplicationJSON
-  openapiH = ctToHeader CTOpenAPI
-  schema = toS $ configSchema conf
+  jsonH = toHeader CTApplicationJSON
   shouldCount = iPreferCount apiRequest
+  schema = toS $ configSchema conf
   topLevelRange = fromMaybe allRange $ M.lookup "limit" $ iRange apiRequest
-  mapSnd f (a, b) = (a, f b)
-  readDbRequest = DbRead <$> buildReadRequest (configMaxRows conf) (dbRelations dbStructure) (map (mapSnd pdReturnType) $ dbProcs dbStructure) apiRequest
-  mutateDbRequest = DbMutate <$> buildMutateRequest apiRequest
-  selectQuery = requestToQuery schema False <$> readDbRequest
-  countQuery = requestToCountQuery schema <$> readDbRequest
-  mutateQuery = requestToQuery schema False <$> mutateDbRequest
-  readSqlParts = (,) <$> selectQuery <*> countQuery
-  mutateSqlParts = (,) <$> selectQuery <*> mutateQuery
-  respondToRange response = if topLevelRange == emptyRange
-                            then return $ errResponse status416 "HTTP Range error"
-                            else response
   rangeHeader queryTotal tableTotal = let lower = rangeOffset topLevelRange
```

Comment:
Even though this function hasn't been modified, it should probably have the let statement on the next line and indented to avoid the extreme indentation.